tx-source/site_core.php

```diff
@@ -1148,13 +1148,27 @@
 $Definition['Revoke'] = 'Revoke';
 $Definition['Revoked'] = 'Revoked';
 $Definition['Revoked badge.'] = 'Revoked badge.';
+$Definition['richEditor.description.title'] = 'Rich Text Editor.';
+$Definition['richEditor.description.paragraphMenu'] = 'To edit a paragraph\'s style, hit tab to get to the paragraph menu. From there you will be able to pick one style. Nothing defaults to paragraph.';
+$Definition['richEditor.description.inlineMenu'] = 'A inline formatting menu will show up when you select text. Hit tab to get into that menu.';
+$Definition['richEditor.menu.blockquote'] = 'Blockquote';
+$Definition['richEditor.menu.bold'] = 'Bold';
+$Definition['richEditor.menu.code'] = 'Inline Code Block';
+$Definition['richEditor.menu.codeBlock'] = 'Code Block';
+$Definition['richEditor.menu.heading'] = 'Heading';
+$Definition['richEditor.menu.inline'] = 'Inline Style Menu';
+$Definition['richEditor.menu.italic'] = 'Italic';
+$Definition['richEditor.menu.link'] = 'Link';
+$Definition['richEditor.menu.paragraph'] = 'Paragraph Style Menu';
+$Definition['richEditor.menu.spoiler'] = 'Spoiler';
+$Definition['richEditor.menu.subheading'] = 'Subheading';
+$Definition['richEditor.menu.strike'] = 'strike';
```

Comment:
We should capitalize this one with the others.

Reply:
Most keys are default text, but there aren't. I based the casing on https://github.com/vanilla/locales/blob/1aea33f2e6da87b20c28653751ab611431a00ab7/tx-source/site_core.php#L813